043-SQLModernization/README.md

```diff
@@ -0,0 +1,60 @@
+# What The Hack - SQL Modernization and Migration
+
+## Introduction
+SQL Server has come a long way!  From OLTP to OLAP, on-premises to cloud, big data clusters to hyperscale -- SQL Server has evolved in many ways.  With that evolution comes complexity as organizations seek to take advantage of new features and cloud efficiency.  This hack puts you (and your team) in the role of a SI (Systems Integrator), working with many clients to modernize their environments and applications.
+
+## Learning Objectives
+In this hack, database administrators, developers who work extensively with SQL Server, and architects will migrate solutions to Azure. Several different scenarios are presented that are based on real-world scenarios, and in most cases, there are multiple ways to solve the problem given the requirements of each challenge.  This hack also prepares participants for the DP-300 exam.  While some challenges will touch on topics such as ETL/SSIS and modernizing data deployment (i.e., data ops), there are What The Hacks that focus extensively on these topics; see related hacks above.
+
+## Challenges
+
+0. Challenge 0: [Setup](./Student/Challenge00.md)
+1. Challenge 1: [Migration](./Student/Challenge01.md)
+2. Challenge 2: [Advanced Migration at Scale](./Student/Challenge02.md)
+3. Challenge 3: [Performance & Tuning](./Student/Challenge03.md)
+4. Challenge 4: [Security & Auditing](./Student/Challenge04.md)
+5. Challenge 5: [High Availability & Disaster Recovery](./Student/Challenge05.md)
+6. Challenge 6: [Data Governance](./Student/Challenge06.md)
+
+
+## Technologies
+1. Azure SQL Database / Managed Instance
+1. Azure Database Migration service
+1. Azure Data Studio
+1. Azure Purview
+
+## Prerequisites
+- Your own Azure subscription with Owner access
+- See the [setup challenge](./Student/Challenge00.md) for information on setting up your environment.
+
+## Repository Contents
+- `../Coach/Solutions`
+  - Coach's Guide for challenges
+  - Solution files for challenges
+- `../Student`
+  - Student's Challenge Guide
+- `../images`
```

Comment:
This is actually in /Coach/images.  I don't think you need to list this here, so I'd just remove this line.